### PR TITLE
Cleaned and refactored code

### PR DESCRIPTION
**Pull request by Pytness**
- Replaced spaces with tabs
- Cleaned code
- Refactored code

### CODE REVIEW

**Comment by Pytness** [comment]
Also, I think it would be cool if the plugin worked with the right click, so you do don't have to throw your seed away every time you want to actually remove a seed.

**Comment by Geolykt** [comment]
I belive that it is wiser to stay with spaces due to some applications and operating systems not fully supporting tabulators

File: src/main/java/io/github/toomanybugs1/QuickFarm/Main.java [hunk 1]
```diff
@@ -24,151 +24,159 @@
...
+		this.mcmmo = (mcMMO) Bukkit.getServer().getPluginManager().getPlugin("mcMMO");
+
+		if (this.mcmmo == null)
+			getLogger().info("This server does not have mcMMO. Disabling mcMMO features.");
+		else
+			getLogger().info("mcMMO has been detected!");
+	}
+
+	@Override
+	public boolean onCommand(@NotNull CommandSender sender, Command cmd, @NotNull String label, String[] args) {
```
Comment (Geolykt) [inline src/main/java/io/github/toomanybugs1/QuickFarm/Main.java]: The annotation is not really required here as it has no noteable effect and not everyone has the library to begin with (yes, I'm a filthy Eclipse User)

File: src/main/java/io/github/toomanybugs1/QuickFarm/Main.java [hunk 2]
```diff
@@ -4,6 +4,9 @@
 import java.util.Collection;
 import java.util.List;
 
+import org.bukkit.*;
```
Comment (Geolykt) [inline src/main/java/io/github/toomanybugs1/QuickFarm/Main.java:6]: I'd rather have all the Imports sorted alphabetically, and not having the Bukkit Imports divided in two chunks

File: src/main/java/io/github/toomanybugs1/QuickFarm/Main.java (hunk 1 above)
Comment (Pytness) [reply src/main/java/io/github/toomanybugs1/QuickFarm/Main.java]: I just changed it to reduce the number of annoying warnings

**Comment by Pytness** [comment]
> I belive that it is wiser to stay with spaces due to some applications and operating systems not fully supporting tabulators

Pretty unlikely in 2020, but I changed it back to spaces in dc59925 anyways

**Comment by toomanybugs1** [comment]
Hey sorry about the wait, been busy with school. Let me test this in a bit and I'll merge it if all is well. Thanks for the contribution!

**Pull request merged**